src/app/[locale]/projects/detail/[id]/components/Attachments.tsx

```diff
@@ -218,7 +226,7 @@ export default function ProjectAttachments({ projectId }: { projectId: string })
 						attachment.checkedTeam ?? '',
 						attachment.checkedBy ?? '',
 						attachment.usageAttachment ?? 'n/a',
-						''
+						{ projectId, attachmentId: attachment['_links']['self']['href'].substring(attachment['_links']['self']['href'].lastIndexOf('/') + 1), attachmentName: attachment.filename }
```

Comment:
This can be re-write as below
```suggestion
						{ projectId, attachmentId: attachment['_links']['self']['href'].split('/').at(-1)), attachmentName: attachment.filename }
```